### PR TITLE
book: add qqmlextensionplugin concept

### PR DESCRIPTION
**Pull request by ahayzen-kdab**
Change-Id: I22e81a80d72335786ab4a8c7dd9cb3de4d85750e

### CODE REVIEW

File: book/src/concepts/qqmlextensionplugin.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!--
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# QQmlExtensionPlugin
+
+Qt allows for plugins containing object definitions to be loaded a runtime from a directory instead of being embedded into the application.
```
Comment (LeonMatthesKDAB) [inline book/src/concepts/qqmlextensionplugin.md:10]: loaded a runtime -> loaded **at** runtime

**Review by LeonMatthesKDAB — COMMENTED** [review]
Found a typo, otherwise looks good.

File: book/src/concepts/qqmlextensionplugin.md (hunk 1 above)
Comment (ahayzen-kdab) [reply book/src/concepts/qqmlextensionplugin.md]: done

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Pull request merged**